frontend/kesaseteli/employer/src/hooks/backend/useLogoutQuery.ts

```diff
@@ -13,6 +14,7 @@ const useLogoutQuery = (): UseMutationResult<unknown, Error, void> => {
     {
       onSuccess: () => {
         void queryClient.removeQueries();
+        clearLocalStorage('application');
```

Comment:
I think the key `application` should be either a constant or strictly typed. Otherwise, any future adjustments will keep the data stored in localStorage.

Reply:
I started to use enum for it. Another place it is in use is here:

Reply:
It's kind of hard because sometimes keys are dynamically created, like 
`clearLocalStorage(`application-${completeApplication.id}`)` It's kinda same as with react-querys invalidateQueries:
https://react-query.tanstack.com/guides/query-invalidation